Exporter/build.gradle

```diff
@@ -55,8 +55,8 @@ dependencies {
     runtime("com.h2database:h2:1.4.196")
 
     testCompile("org.springframework.boot:spring-boot-starter-test")
-    testCompile("com.jamesmurty.utils:java-xmlbuilder:1.2")
     testCompile("org.mockftpserver:MockFtpServer:2.0.2")
     testCompile("org.json:json:20171018")
+    testCompile("com.jamesmurty.utils:java-xmlbuilder:1.2")
```

Comment:
Do we still need this?

Reply:
yes it use for unit testing xml generated files.